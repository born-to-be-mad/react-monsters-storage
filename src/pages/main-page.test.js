import { shallow } from "enzyme";
import React from "react";
import MainPage from "./main-page.component";

test("expect to render MainPage component - snapshot", () => {
  const wrapper = shallow(<MainPage />);
  expect(wrapper).toMatchSnapshot();
});
