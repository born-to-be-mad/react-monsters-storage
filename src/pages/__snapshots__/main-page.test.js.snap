// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render MainPage component - snapshot 1`] = `
<div
  className="MainPage"
>
  <Header />
  <SearchBox
    handleChange={[Function]}
    placeholder="Enter search criteria"
  />
  <Scroll>
    <h1>
      Loading
    </h1>
  </Scroll>
</div>
`;
